### DISCUSSION TITLE
Why are there no releases schedule for selfhosted ?

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#2189] [Self-Hosted Support]
What can we as a community around plausible do to make the releases happen as planned? 

And can we even talk about if it makes sense to release quarterly or monthly?

What is holding back?

**Comment by ukutaht** [comment]
All of our engineering effort since May or so has gone to ensuring platform stability for Plausible.Cloud. Sorry about the delays.

@ruslandoga will start as our Open Source Developer on Sept 12th and we are focused on delivering `v1.5.0` as the first task. The core team cannot currently focus on this, but the work on self-hosted will continue next week.

Once we have someone on the team that owns these releases, we can maybe talk about different schedules.

**Comment by ruslandoga** [comment]
👋

[`v1.5.0-rc.0`](https://github.com/plausible/analytics/discussions/2324) is out! Please try it out :)
